nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -643,7 +647,14 @@ export function RanEdit(props: Props) {
     props.gateway?.connected_enodeb_serials ||
       DEFAULT_GATEWAY_CONFIG.connected_enodeb_serials,
   );
-
+  const menuProps = {
```

Comment:
minor nit: why aren't we using class here(useStyles) pattern  here ?

Reply:
forgot to change it :)